File: Add Code Here/LeetCode_Sols/Easy/557. Reverse Words in a String III/solution.java
class Solution {
    public String reverseWords(String s) {
        StringBuilder res = new StringBuilder();
        String[] arr = s.split(" ");
        for (String str : arr){
            res.append(new StringBuffer(str).reverse().append(" "));
        }
        
       // res.deleteCharAt(res.length() - 1);   If you don't want to use trim()
        return res.toString().trim();        
    }
}  

File: Add Code Here/LeetCode_Sols/Easy/557. Reverse Words in a String III/solution.dart
class Solution {
  String reverseWords(String s) {
      String rev = "";
      List words = s.split(" ");
      for (int i = 0 ; i< words.length; i++){
          rev = rev + words[i].split("").reversed.join() + " ";
          }
          
          return rev.trim();
  }
}